docs/EXAMPLES.md

```diff
@@ -14,8 +14,8 @@
 
 ## setTimeout
 
-| 1 optional argument                     |
-|-----------------------------------------|
+|           1 optional argument           |
```

Comment:
You didn't necessarily have to change it back. I'm fine with both ways. It's on you!
I just asked you why you changed it? By the way, I'm still curious about that. 😜

Reply:
I just formatted using a markdown plugin 🤷‍♂️  that's why I prefer to revert back 😂